apps/src/lab2/responseValidators.ts

```diff
@@ -30,20 +30,20 @@ const BlocklySourceResponseValidator: ResponseValidator<
   return sourceValidatorHelper(response, blocklyValidator);
 };
 
-// Validator for Python sources.
-const PythonSourceResponseValidator: ResponseValidator<
+// Validator for CodeBridge sources.
+const CodeBridgeSourceResponseValidator: ResponseValidator<
```

Comment:
@molly-moen @thomasoniii made the validator updates here, let me know if something doesn't look right!

Reply:
`Codebridge` should be all one word (instead of `CodeBridge`)

Reply:
Done 👍